Assets/_BForBoss/_Weapons/Scripts/CrosshairBehaviour.cs

```diff
@@ -0,0 +1,28 @@
+using Sirenix.OdinInspector;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace Perigon.Weapons
+{
+    public class CrosshairBehaviour : MonoBehaviour
+    {
+        [SerializeField] private Sprite _defaultCrosshair = null;
+        private Image _crosshair = null;
+        
+        private void Awake()
+        {
+            _crosshair = GetComponentInChildren<Image>();
+        }
+
+        private void Start()
+        {
+            _crosshair.sprite = _defaultCrosshair;
+        }
+
+        [Button]
+        public void SetCrosshairImage(Sprite image)
+        {
+            _crosshair.sprite = image;
+        }
```

Comment:
Is the point of this just so we can quickly test different sprites?

Reply:
yup